Description: Use gcc instead of g++ for linking
 Upstream uses g++ to link the final buckle binary, which is
 not necessary and adds an unneeded dependency to libstdc++.
Author: Dominik George <nik@naturalnet.de>
Bug: https://github.com/zevv/bucklespring/issues/22
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ endif
 
 OBJS    = $(subst .c,.o, $(SRC))
 CC 	= $(CROSS)gcc
-LD 	= $(CROSS)g++
+LD 	= $(CROSS)gcc
 STRIP 	= $(CROSS)strip
 
 %.o: %.c
